common/jwt.py

```
@@ -0,0 +1,35 @@
+        settings.SECRET_KEY,
+        algorithm=settings.ALGORITHM
+    )
+    return encoded_jwt
+
+def decode_access_token(token: str) -> str | None:
+    try:
+        payload = jwt.decode(token,settings.SECRET_KEY,algorithms=[settings.ALGORITHM])
+        email: str= payload.get("sub")
+        if not email:
+            raise HTTPException(status_code=status.HTTP_404_NOT_FOUND,detail="사용자 정보가 없습니다.")
+        return email
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)